templates/blog/author.html

```diff
@@ -4,67 +4,64 @@
 
 {% block description %}{% if author.id == 217 %}Canonical produces Ubuntu, provides commercial services for Ubuntu's users, and works with hardware manufacturers, software vendors and cloud partners to certify Ubuntu.{% else %}{{ author.description|safe }}{% endif %}{% endblock %}
 
+{% set is_paper = True %}
+
 {% block content %}
-<div class="p-strip--image is-deep" style="background-image: url(https://assets.ubuntu.com/v1/f8a323a7-image-background-paper.png);">
-  <div class="row">
-    <div class="col-8">
-      <div class="p-media-object u-no-margin--bottom">
-        <img src="{% if author.id == 217 %}https://assets.ubuntu.com/v1/60d9b81e-picto-canonical.svg{% else %}{{ author.avatar_urls['96'] }}{% endif %}" class="p-media-object__image is-round" alt="">
 
-        <div class="p-media-object__details">
-          <h1 class="p-media-object__title">
-            {{ author.name }}
-          </h1>
-          {% if author.user_job_title %}
-            <p class="p-media-object__content"><em>{{ author.user_job_title }}</em></p>
+<section class="p-strip">
+    <div class="row p-section">
+      <div class="col-3 col-medium-2 u-hide--small">
+        <div class="p-image-wrapper">
+          {% if author.id != 217 %}
+            <div class="p-image--2-3" style="background-image: url({{ author.avatar_urls['96'] }}); max-width: 4rem"></div>
           {% endif %}
-
+        </div>
+      </div>
+      <div class="col-6 col-start-large-4 col-medium-4 col-start-medium-3 col-small-4">
+          <h1>{{ author.name }}</h1>
```

Comment:
This page is missing the breadcrumb so it's difficult to navigate back to the blog